extensions/jaeger/runtime/src/main/java/io/quarkus/jaeger/runtime/JaegerDeploymentTemplate.java

```diff
@@ -50,15 +49,9 @@ public void registerTracer(JaegerConfig jaeger) {
     private boolean isValidConfig(JaegerConfig jaeger) {
         Config mpconfig = ConfigProvider.getConfig();
         Optional<String> serviceName = mpconfig.getOptionalValue(JAEGER_SERVICE_NAME, String.class);
```

Comment:
It's not related to this issue but why do you do that and get the MP config explicitly? Isn't the Quarkus config sufficient? Or you want to support the default configuration with MP Config too?

Reply:
Yeah want to support the default configuration approach aswell.